From daf8d0e5a3d0b1d619627238b0985002afd81b40 Mon Sep 17 00:00:00 2001
From: Naphtali Sprei <nsprei@redhat.com>
Date: Wed, 7 Apr 2010 16:44:14 -0300
Subject: [PATCH 03/21] read-only: Added drives' readonly option

RH-Author: Naphtali Sprei <nsprei@redhat.com>
Message-id: <1270658667-14294-4-git-send-email-nsprei@redhat.com>
Patchwork-id: 8438
O-Subject: [RHEL6 kvm PATCH v3 rebase 03/16] read-only: Added drives' readonly
	option
Bugzilla: 537164
RH-Acked-by: Daniel P. Berrange <berrange@redhat.com>
RH-Acked-by: Christoph Hellwig <chellwig@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=537164
upstream commit: 3f3ed5933aa9b7890d9ba5bbc418d153aa61a5f7

Signed-off-by: Naphtali Sprei <nsprei@redhat.com>
---
 qemu-options.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-options.hx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index b35e4ed..5efb57b 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -104,7 +104,7 @@ DEF("drive", HAS_ARG, QEMU_OPTION_drive,
     "       [,cyls=c,heads=h,secs=s[,trans=t]][,snapshot=on|off]\n"
     "       [,cache=writethrough|writeback|none][,format=f][,serial=s]\n"
     "       [,addr=A][,id=name][,aio=threads|native]\n"
-    "       [,boot=on|off]\n"
+    "       [,boot=on|off][,readonly=on|off]\n"
     "                use 'file' as a drive image\n")
 DEF("set", HAS_ARG, QEMU_OPTION_set,
     "-set group.id.arg=value\n"
-- 
1.7.0.3

